### PR TITLE
Updates to main page template, css

### PR DESCRIPTION
**Pull request by Andrew-Rendler**
This has the addition of 6 boxes, for current price, avg price, $/% change, Market Cap, and Volume. We can pretty easily incorporate the work done by Ryan earlier today, to display the actual values in these placeholders.

### CODE REVIEW

File: HTML/index.html [hunk 1]
```diff
@@ -33,12 +33,120 @@
       </div>
 
       <div class="row">
-        <div class="col-sm-8">
-          <img src="math.png" alt="damn damn" width=100% height=40%/>
+	  
+	  
+	          <div class="col-sm-4">
+		<div class="Current">
+          <table style="width:350px; float: center; text-align: center;">
```
Comment (netse-ai) [inline HTML/index.html:40]: Small changes needed:

Let's keep html classes lowercased: `Current` -> `current`.

Another thing is avoiding inline styles. the `table` should be given a class name and the styles then moved to the css file.

File: HTML/index.html [hunk 2]
```diff
@@ -33,12 +33,120 @@
...
+			<th > <h1>Market Cap</h1> </th>
+			</tr>
+			<tr>
+			<td> <h2>100M</h2></td>
+			</tr>
+			</table>
+			</div>
+        </div>
+		
+	<!-- Example sql query to populate Current Price table, and other metrics in math section.
```
Comment (netse-ai) [inline HTML/index.html:128]: Nice, we will integrate this into our external api endpoints when the database is up.

**Review by netse-ai — CHANGES_REQUESTED** [review]

**Comment by powerryan** [comment]
LGTM

**Review by powerryan — APPROVED** [review]

**Comment by BenKupernik** [comment]
Cool, lgtm

**Pull request closed without merging**